service/migration_manager.cc

```diff
@@ -109,6 +109,14 @@ void migration_manager::init_messaging_service()
         _feature_listeners.push_back(_feat.cdc.when_enabled(update_schema));
         _feature_listeners.push_back(_feat.per_table_partitioners.when_enabled(update_schema));
         _feature_listeners.push_back(_feat.computed_columns.when_enabled(update_schema));
+
+        if (!_feat.table_digest_insensitive_to_expiry) {
+            _feature_listeners.push_back(_feat.table_digest_insensitive_to_expiry.when_enabled([this] {
+                (void) with_gate(_background_tasks, [this] {
+                    return reload_schema();
```

Comment:
So is the goal to reload the mutations just to re-run them through the digester? If we generated the mutations from the schema object, could be have avoided it (don't remember if we have such an ability).

Reply:
We could generate digests from schema_ptr in memory. The reason I go through the schema merge path is to make sure that this process is synchronized with other schema changes which could be happening concurrently, and that I reuse the existing mechanism for redistributing schema_ptrs. It's less efficient but on the safe side. Having a single path means less risk due to the two paths diverging over time.